src/test/java/mate/academy/dao/RoleDaoTest.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.dao;
+
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import mate.academy.dao.impl.RoleDaoImpl;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoTest extends AbstractTest {
+    private RoleDao roleDao;
+    private Role adminRole;
+    private Role userRole;

```

Comment:
make these variables local, because you use them only in one method. Look through other classes and check if you can do the same with other fields